test/sanity/classes/birds.wtest

```diff
@@ -0,0 +1,15 @@
+class Golondrina {
+	var property energia = 50
+	method volar(kms) {
+		energia -= kms
+	}
+}
+
+test "when a swallow flies it does not affect the others" {
+  const pepita = new Golondrina()
+	assert.equals(50, pepita.energia())
+	const pepona = new Golondrina()
+	pepona.volar(2)
+	assert.equals(48, pepona.energia())
+	assert.equals(50, pepita.energia())
+}
```

Comment:
I'm not sure about this kind of test. It's not very clear what the testing subject is. What about a model-agnostic description like "changing the state of an object does not affect other instances of the same class". We could place that in one of the other files.
I'm just saying this because if the "birds.wtest" fails it's not really going to be obvious what is wrong, and might be hard to prevent the future addition of a similar test with some other domain.

Reply:
Changed the name of the test.